cmd/syncProcessor.go

```diff
@@ -49,7 +54,7 @@ func newSyncTransferProcessor(cca *cookedSyncCmdArgs, numOfTransfersPerPart int,
 
 		// flags
 		BlobAttributes: common.BlobTransferAttributes{
-			PreserveLastModifiedTime: true, // must be true for sync so that future syncs have this information available
+			PreserveLastModifiedTime: preserveLastModifiedTime,
```

Comment:
Is it equivalent to: `!cca.preserveSMBInfo`?

Reply:
Oh yeah, it is!

Reply:
It would be equivalent to `cca.preserveSMBInfo` which makes it simpler. Good call out :)